modules/mssql/README.md

```diff
@@ -15,7 +15,7 @@ The following dependency must be available for SQL Server module:
 | additional\_users | A list of users to be created in your cluster. A random password would be set for the user if the `random_password` variable is set. | <pre>list(object({<br>    name            = string<br>    password        = string<br>    random_password = bool<br>  }))</pre> | `[]` | no |
 | availability\_type | The availability type for the master instance.This is only used to set up high availability for the MSSQL instance. Can be either `ZONAL` or `REGIONAL`. | `string` | `"ZONAL"` | no |
 | backup\_configuration | The database backup configuration. | <pre>object({<br>    binary_log_enabled             = bool<br>    enabled                        = bool<br>    point_in_time_recovery_enabled = bool<br>    start_time                     = string<br>    transaction_log_retention_days = string<br>    retained_backups               = number<br>    retention_unit                 = string<br>  })</pre> | <pre>{<br>  "binary_log_enabled": null,<br>  "enabled": false,<br>  "point_in_time_recovery_enabled": null,<br>  "retained_backups": null,<br>  "retention_unit": null,<br>  "start_time": null,<br>  "transaction_log_retention_days": null<br>}</pre> | no |
-| create\_timeout | The optional timeout that is applied to limit long database creates. | `string` | `"15m"` | no |
+| create\_timeout | The optional timeout that is applied to limit long database creates. | `string` | `"30m"` | no |
```

Comment:
It looks like `make generate_docs` wasn't run when this default value was changed in https://github.com/terraform-google-modules/terraform-google-sql-db/pull/425 - might be a good idea to add a CI check for this

Reply:
Raised this as a separate PR if we want to keep the diff clean on this one https://github.com/terraform-google-modules/terraform-google-sql-db/pull/440

Reply:
#440 was merged, so the noise in the diff is gone now